backend/src/services/hsm/hsm-fns.ts

```diff
@@ -0,0 +1,40 @@
+import * as grapheneLib from "graphene-pk11";
+
+import { getConfig } from "@app/lib/config/env";
+import { logger } from "@app/lib/logger";
+
+export type HsmModule = {
+  module: grapheneLib.Module | null;
```

Comment:
I think this should be moved out of hsm-fns and keep it in ``hsm-module` or something. 

I was searching for this file as hsm-module 😅